html/elements/audio.json

```diff
@@ -407,10 +407,12 @@
           "__compat": {
             "support": {
               "chrome": {
-                "version_added": "3"
+                "version_added": "3",
+                "notes": "Defaults to <code>metadata</code> in 64."
```

Comment:
Could you phrase this as "version 64" or "Chrome 64"? I think it's a little confusing without and I couldn't find any other examples of a version number being used like this on first mention.

This applies throughout the diff.